test/APITests.js

```diff
@@ -69,6 +71,20 @@ describe('API Tests', () => {
       // expect(Object.keys(component.viewHeaders).length).toBeGreaterThan(0);
     });
 
+    it('creates an API using a promise', (done) => {
+      div = global.document.createElement('div');
+      global.document.body.appendChild(div);
+
+      div.setAttribute('style', 'width:600px; height: 400px; background-color: lightgreen');
+
+      const p = viewer(div, simpleCenterViewConfig, undefined, true);
+      p.then((hgApi) => {
+        expect(hgApi).not.toEqual(null);
+        api = hgApi;
```

Comment:
Is this line needed for anything? If so please ignore my suggestion. Otherwise, I think we can remove it.
```suggestion
```